examples/00-load/create-point-cloud.py

```diff
@@ -26,14 +26,17 @@
 def generate_points(subset=0.02):
     """A helper to make a 3D NumPy array of points (n_points by 3)"""
     dataset = examples.download_lidar()
-    ids = np.random.default_rng().integers(
+    ids = rng.integers(
         low=0,
         high=dataset.n_points - 1,
         size=int(dataset.n_points * subset),
     )
     return dataset.points[ids]
 
 
+# Seed the random number generator for generating data
+rng = np.random.default_rng(seed=0)
```

Comment:
It is confusing to me to define this after the function that references it, although it is valid code.  I would recommend putting this up top.

Reply:
Moved it up in https://github.com/pyvista/pyvista/pull/6106/commits/f155577032d248d99e9f0c5abfa7af9fc4152e2f, hopefully that's clearer now